crates/database/src/settlement_observations.rs

```diff
@@ -31,15 +31,15 @@ SET gas_used = $1, effective_gas_price = $2, surplus = $3, fee = $4
 
 pub async fn fetch(
     ex: &mut PgConnection,
-    tx_hash: &TransactionHash,
-) -> Result<Option<Observation>, sqlx::Error> {
+    tx_hashes: &[TransactionHash],
+) -> Result<Vec<Observation>, sqlx::Error> {
```

Comment:
Looks like the table is used only for debugging and never queried outside the e2e tests, right? How about specifying this right in the docs?

Reply:
Do you mean in the database readme or?

Reply:
Yep, in the readme.